package.json

```diff
@@ -53,16 +53,15 @@
     "generate": "export SHA=f4424fc69ad29e31110d711310df644a57fee314 && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
     "generate-local": "export REMOTE=../openapi/ && yarn generate-meta",
     "generate-meta": "if [ -z \"${CLOUD_URL}\" ]; then yarn generate-meta-oss; else yarn generate-meta-cloud; fi",
-    "generate-meta-oss": "yarn oss-api && yarn notebooks && yarn unity && yarn flows && yarn managedFunctions && yarn annotations-oss",
-    "generate-meta-cloud": "yarn cloud-api && yarn notebooks && yarn unity && yarn flows && yarn managedFunctions && yarn annotations",
```

Comment:
lol what a weird diff log

Reply:
lol right?? Had to read through it a couple times to make sure the change actually makes sense 😆